lib/iris/tests/unit/coords/test_Coord.py

```diff
@@ -323,5 +323,30 @@ def test_convert_unknown_units(self):
         with self.assertRaisesRegexp(UnitConversionError, emsg):
             coord.convert_units('degrees')
 
+
+class Test___str__(tests.IrisTest):
+
+    def test_short_time_interval(self):
+        coord = DimCoord([5], standard_name='time',
+                         units='days since 1970-01-01')
+        expected = ("DimCoord([1970-01-06 00:00:00], standard_name='time', "
+                    "calendar='gregorian')")
+        result = coord.__str__()
```

Comment:
Out of interest - why not use the built-in ``str``?

Reply:
To catch up an offline conversation – the reason is that in this test class I'm testing the literal `coord.__str__` implementation. While this and `str` are of course behaviourally identical, using `__str__` here seemed a more explicit statement of intent than using `str`.